tools/clang/test/CodeGenHLSL/literals_Mod.hlsl

```diff
@@ -97,6 +102,8 @@ float test() :Foo {
   VERIFY_TYPES(uint, 2 * 2UL);
   VERIFY_TYPES(int64_t, 2 * 2LL);
   VERIFY_TYPES(uint64_t, 2 * 2ULL);
+  VERIFY_TYPES(half, 1.0 * 2 * 1.5h);
+  VERIFY_TYPES(half, 1.0 * 2 * 1.5H);
```

Comment:
Same here, half will implicitly promote to float once supported, not the other way around.